runtime/src/parachains.rs

```diff
@@ -238,34 +174,12 @@ decl_storage! {
 		///	second if the total length (in bytes) of the message payloads.
 		pub RelayDispatchQueueSize: map ParaId => (u32, u32);
 
-		/// Did the parachain heads get updated in this block?
-		DidUpdate: bool;
+		/// The ordered list of ParaIds that have a `RelayDispatchQueue` entry.
+		NeedsDispatch: Vec<ParaId>;
```

Comment:
more documentation here about under what circumstances it is expected not to have a `RelayDispatchQueue` entry would be helpful

Reply:
from the existing docs:

> Messages ready to be dispatched onto the relay chain.

therefore, this is the ordered list of para ids that have "messages ready to be dispatched onto the relay chain".

not being in here therefore means that the para has no "messages ready to be dispatched onto the relay chain".

is that clear?